tcptracer-bpf.c

```diff
@@ -768,7 +768,7 @@ int kretprobe__inet_csk_accept(struct pt_regs *ctx)
 	}
 
 	struct tracked_socket t = {.active = 1};
-	t.prev_send_time_ns = bpf_ktime_get_ns();
+	t.prev_receive_time_ns = bpf_ktime_get_ns();
```

Comment:
Why do we update this value in `kretprobe__inet_csk_accept` and also `kprobe__skb_copy_datagram_iter`?

Reply:
`tcp_recvmsg` kprobe wouldn't be an option?

Reply:
Just in case if skb_c.d.i. hook will be missing, to not calculate latency as from OS start (or whatever ktime clock counts). Here it's simply defensive programming

Reply:
There is no `tcp_recvmsg` only `tcp_v4_rcv` and `tcp_v6_rcv`.  The skb_cdi is the common point for both